auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java

```diff
@@ -59,6 +59,19 @@ public void cleanupVirtualGroups(String accountId, String environmentCrn) {
         }
     }
 
+    public void assignEnvironmentAdminRole(String userCrn, String environmentCrn) {
+        try {
+            grpcUmsClient.assignResourceRole(userCrn, environmentCrn, grpcUmsClient.getBuiltInEnvironmentAdminResourceRoleCrn(), MDCUtils.getRequestId());
+            LOGGER.debug("EnvironmentAdmin role of {} environemnt is assigned to the {} user", environmentCrn, userCrn);
+        } catch (StatusRuntimeException ex) {
+            if (Code.ALREADY_EXISTS.equals(ex.getStatus().getCode())) {
```

Comment:
I didn't know about this code, I would move this exception handling into GrpcUmsClient assignResourceRole method

Reply:
I would  left this code here as it is specific during environment creation, maybe later for other resources it could be an error case.

Reply:
ok, it is fine for me